src/parse.rs

```diff
@@ -4,7 +4,7 @@ use std::str::{FromStr, from_utf8, from_utf8_unchecked};
 
 use de::{Error, ParseError, Result};
 
-const DIGITS: &[u8] = b"0123456789";
+const DIGITS: &[u8] = b"0123456789ABCDEF";
```

Comment:
Should we support lowercase characters?

Reply:
Since all the language's syntaxes do, I think we should as well